packages/cli/src/WorkflowRunner.ts

```diff
@@ -54,12 +56,48 @@ export class WorkflowRunner {
 		private readonly workflowStaticDataService: WorkflowStaticDataService,
 		private readonly nodeTypes: NodeTypes,
 		private readonly permissionChecker: PermissionChecker,
+		private readonly ownershipService: OwnershipService,
 	) {
 		if (this.executionsMode === 'queue') {
 			this.jobQueue = Container.get(Queue);
 		}
 	}
 
+	async init() {
+		if (this.executionsMode === 'queue') return;
+
+		const executionIds = await this.executionRepository.getNewExecutionIds();
+		for (const executionId of executionIds) {
+			this.logger.debug('Resuming enqueued execution', { executionId });
```

Comment:
But we're re-enqueuing, not necessarily resuming. Maybe this log needs to be at the concurrency service.